README.md

```diff
@@ -1,10 +1,16 @@
 # cross-section-link-report
 Script to generate report about the cross sectional links used in course
 
-1. Create a new file called `config.yml` based on `config.yml.example`. Populate all the variables there.
+1. Checkout the script in some directory in your OpenEdx instance like so:
+```
+export SCRIPT_DIR='/edx/app/edxapp/script'
+git clone https://github.com/open-craft/cross-section-link-report.git  $SCRIPT_DIR/cross-section-link-report
+cd $SCRIPT_DIR/cross-section-link-report/
```

Comment:
There are few additional steps while running this on the production server - i.e., you can clone this into the `/edx/app/edxapp/` directory as an `edxapp` user, but you need to run the `python` command as `www-data`.

That's because only `edxapp` can create directories in `/edx/app/edxapp`, and only `www-data` can create directories in `/edx/var/edxapp/data/`.

We can add a quick note about this to the README.

Reply:
Done